app/models/folder/Folder.scala

```diff
@@ -133,10 +137,16 @@ class FolderDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
   protected def isDeletedColumn(x: Folders): Rep[Boolean] = x.isdeleted
 
   protected def parse(r: FoldersRow): Fox[Folder] =
-    Fox.successful(Folder(ObjectId(r._Id), r.name))
+    for {
+      metadata <- JsonHelper.parseAndValidateJson[JsArray](r.metadata.getOrElse("[]")).toFox
```

Comment:
this might already be worth a one-liner helper function `private def parseMetadata(literal: Option[String]): JsArray)` :)